src/imitation/scripts/train_adversarial.py

```diff
@@ -140,26 +167,20 @@ def train(
         max_episode_steps=max_episode_steps,
     )
 
-    # TODO(shwang): Let's get rid of init_rl later on?
-    # It's really just a stub function now.
-
     # if init_tensorboard:
     #     tensorboard_log = osp.join(log_dir, "sb_tb")
     # else:
     #     tensorboard_log = None
 
+    # TODO(shwang): Let's get rid of init_rl later on?
+    # It's really just a stub function now.
```

Comment:
AFAICT the only reason why we still have `init_rl()` is because it has a hack to get around SB3 issue #109. Maybe merge this comment with my comment below to make it clear that we need to do: (fix 109) -> (remove init_rl)?

Reply:
There seem to be two separate problems here, only one of which is related to #109.

The first is that Sacred transformed dict into `ReadOnlyDict` which errors on edit within Stable Baselines.

The second is that Stable Baselines can reconfigure our custom logger leading to problems (not immediately clear to me which problem would occur, maybe it would just overwrite our logger).
![image](https://user-images.githubusercontent.com/1750835/94880246-3da64d80-0417-11eb-9441-08e620a92c95.png)

Reply:
It doesn't seem to me that init_rl contains any hack to mitigate the first problem with ReadOnlyDict, unless I'm mistaken that the first and second problem are mostly unrelated.

Reply:
I was referring to issue 109 in the SB3 repo, not in this repo.